views/templates/block/checklist.twig

```diff
@@ -145,30 +145,92 @@
                         {% endif %}
                     </td>
                 </tr>
+                {% if not checkPhpVersionCompatibility %}
+                  <tr>
+                    <td>{{ 'Your current PHP version isn\'t compatible with your PrestaShop version.'|trans }}</td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if not checkApacheModRewrite %}
+                  <tr>
+                    <td>{{ 'Apache mod_rewrite is disabled.'|trans }}</td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if notLoadedPhpExtensions|length > 0 %}
+                  <tr>
+                    <td>
+                      {% if notLoadedPhpExtensions|length > 1 %}
+                        {{ 'The PHP extension %s are not installed.'|trans([notLoadedPhpExtensions|join(', ')]) }}
+                      {% else %}
+                        {{ 'The PHP extension %s is not installed.'|trans([notLoadedPhpExtensions|join(', ')]) }}
+                      {% endif %}
```

Comment:
I think there is a plural option for the trans method?

Reply:
transchoice

Reply:
I saw recently transchoice being forbidden in PrestaShop/PrestaShop
Should we avoid using it everywhere?

Reply:
I saw recently transchoice being forbidden in PrestaShop/PrestaShop
Should we avoid using it everywhere?

Reply:
Here it is https://github.com/PrestaShop/PrestaShop/pull/24622#issuecomment-874582137

Reply:
I had forgotten [about that](https://symfony.com/doc/4.4/translation.html#message-format)

Anyway, transChoice isn't even implemented in the [autoupgrade's translator component](https://github.com/PrestaShop/autoupgrade/blob/dev/classes/UpgradeTools/Translator.php).

Perhaps this can be improved by using a generic wording:

	The following PHP extensions aren't installed: %s

Reply:
Fixed with the last idea of @eternoendless